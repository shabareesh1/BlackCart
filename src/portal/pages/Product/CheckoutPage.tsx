import * as React from 'react';
import { GlobalContainer  , Product} from '../GlobalContainer';
import { withRouter, RouteComponentProps } from 'react-router';
import { withContext } from '../../components/WithContext';
import '../../../style.scss';
import Cart from "../../components/Cart";



interface IProps {
  container: GlobalContainer;
}

interface IState {

    Sizechecked : string
    cardtClicked : boolean

}

class CheckoutPage extends React.Component<IProps & RouteComponentProps<{ viewId: string }>, IState> {
  constructor(props: IProps & RouteComponentProps<{ viewId: string }>) {
    super(props);
    this.state = {
        Sizechecked :'7',
        cardtClicked : false
    };
  }

  componentDidMount() {
  }

    handleChange = (i : any) =>{
        const id = i;
        setTimeout(() => {
            const element: HTMLElement = document.getElementById(id) as HTMLElement;
            if (element !== null && element !== undefined) {
                element.scrollIntoView({ behavior: "smooth" });
            }
        }, 100);


    }

    handleCart  =() =>{

       this.setState({
           ...this.state , cardtClicked : !this.state.cardtClicked
       } )
    }

    onValueChange = (event : any) => {
        this.setState({
          ...this.state,   Sizechecked: event.target.value
        } ,()=>{

           // console.log(this.state.Sizechecked)
        });
    }

    AddtoCart = () =>{
        const item ={
            id : Math.random(),
            title : 'CARPI TRIPLE WHITE' ,
            Category : 'Mens ' ,
            url: './image1.jpg',
          Price : 256,
            Size : this.state.Sizechecked
        } as Product

        this.props.container.AddProducttoCOntainer(item)
        this.handleCart()
    }

  render() {

    return (
      <div>
        <div className="main  ">
          <div>
              <nav className="navbar navbar-expand-lg navbar-light bg-light" style={{backgroundColor:'#fffff'}}>
                  <button className="navbar-toggler" type="button" data-toggle="collapse"
                          data-target="#navbarTogglerDemo03" aria-controls="navbarTogglerDemo03" aria-expanded="false"
                          aria-label="Toggle navigation">
                      <span className="navbar-toggler-icon"></span>
                  </button>

                  <a className="navbar-brand" href="#" style={{ fontWeight:700 , fontSize:'30px'}}>KOIO</a>
                  <div className="collapse navbar-collapse" id="navbarTogglerDemo03">
                      <ul className="navbar-nav mr-auto mt-2 mt-lg-0">
                          <li className="nav-item active">
                              <a className="nav-link" href="#">Shop men <span className="sr-only">(current)</span></a>
                          </li>
                          <li className="nav-item">
                              <a className="nav-link" href="#"> Shop Women</a>
                          </li>
                          <li className="nav-item">
                              <a className="nav-link " href="#">About </a>
                          </li>
                      </ul>

                      <div className="form-inline my-2 my-lg-0">
                          <ul className="navbar-nav mr-auto mt-2 mt-lg-0">
                              <li className="nav-item ">
                                  <a className="nav-link" href="#"> Locations <span className="sr-only">(current)</span></a>
                              </li>
                              <li className="nav-item">
                                  <a className="nav-link" href="#">  CraftmanShip</a>
                              </li>
                              <li className="nav-item"  onClick={this.handleCart}>
                                  <a className="nav-link " href="#"> Cart ({this.props.container.state.Products.length}) </a>
                              </li>
                          </ul>
                      </div>
                  </div>
              </nav>
              {this.state.cardtClicked && <Cart  prop={this.handleCart} />  }
          </div>
            <div className=" Content container-fluid">
                <div className={'row'} style={{height :"100%"}} >
                <div className={'col Productimages'}>
                    <div  className=' row'>
                        <div className={"col"}>
                            <div>
                                <img className="image" onClick={() => this.handleChange('0')} src="./image1.jpg" />
                            </div>
                            <div>
                                <img className="image" onClick={() => this.handleChange('1')} src="./image2.png.jpg" />
                            </div>
                            <div>
                                <img className="image"  onClick={() => this.handleChange('2')} src="./image3.png" />
                            </div>
                            <div>
                                <img className="image"   onClick={() => this.handleChange('0')} src="./image1.jpg" />
                            </div>
                            <div>
                                <img className="image"  onClick={() => this.handleChange('1') } src="./image2.png.jpg" />
                            </div>
                        </div>

                        <div className={"col"}>
                            {/*
                                <img  className={"imageStyle"} src={`${this.state.Selectedimage==='0' ? './large1.png' :
                                    this.state.Selectedimage==='1' ? './large2.png' : "./large3.png" 
                                }`}  />
                                */}
                            <img  className={"imageStyle"} id="0" src ="./large1.png" />
                            <img  className={"imageStyle"} id="1" src ="./large2.png" />
                            <img  className={"imageStyle"} id="2" src ="./large3.png "/>
                        </div>

                    </div>


                </div>
                <div className={'col Productdetails'}>
                    <header id = 'product-header'>
                    <span>
                        Men's
                    </span>
                    <div>
                        CAPRI TRIPLE WHITE
                    </div>
                        <span className="product-pricing">
          <span className="product-price">$268</span>

      </span>
                    </header>

                    <div
                        className="product-option-row product-option-row--productes product-option-row--7 product-option-row--size ">
                        <h5 className="product-header">Size:</h5>


                        <div className="product-element product-element--7-us-40-eu">
                            <input type="radio" name="single-option-select-1" id="product-1-7-us-40-eu"
                                   value="7 " data-index="option1" className="single-option-select" checked={this.state.Sizechecked === "7"}
                                   onChange={this.onValueChange} />
                                <label htmlFor="product-1-7-us-40-eu" data-size="7" tabIndex={0}
                        style={{ border: `${this.state.Sizechecked === "7" ? '2px solid black' : 'none'}` }} >7</label>
                </div>

                        <div className="product-element product-element--8-us-41-eu">
                            <input type="radio" name="single-option-select-1" id="product-1-8-us-41-eu"
                                   value="8 " data-index="option1" className="single-option-select" checked={this.state.Sizechecked == "8"}
                                   onChange={this.onValueChange}   />
                                <label htmlFor="product-1-8-us-41-eu" data-size="8" tabIndex={0}  style={{ border: `${this.state.Sizechecked === "8" ? '2px solid black' : 'none'}` }}>8</label>
                        </div>


                        <div className="product-element product-element--9-us-42-eu">
                            <input type="radio" name="single-option-select-1" id="product-1-9-us-42-eu"
                                   value="9 " data-index="option1" className="single-option-select" checked={this.state.Sizechecked === "9"}
                                   onChange={this.onValueChange}   />
                                <label htmlFor="product-1-9-us-42-eu" data-size="9" tabIndex={0}  style={{ border: `${this.state.Sizechecked === "9" ? '2px solid black' : 'none'}` }}>9</label>
                        </div>


                        <div className="product-element product-element--10-us-43-eu">
                            <input type="radio" name="single-option-select-1" id="product-1-10-us-43-eu"
                                   value="10 " data-index="option1" className="single-option-select"checked={this.state.Sizechecked === "10"}
                                   onChange={this.onValueChange}   />
                                <label htmlFor="product-1-10-us-43-eu" data-size="10" tabIndex={0}  style={{ border: `${this.state.Sizechecked === "10" ? '2px solid black' : 'none'}` }}>10</label>
                        </div>


                        <div className="product-element product-element--11-us-44-eu">
                            <input type="radio" name="single-option-select-1" id="product-1-11-us-44-eu"
                                   value="11 " data-index="option1" className="single-option-select" checked={this.state.Sizechecked === "11"}
                                   onChange={this.onValueChange}  />
                                <label htmlFor="product-1-11-us-44-eu" data-size="11" tabIndex={0}  style={{ border: `${this.state.Sizechecked === "11" ? '2px solid black' : 'none'}` }}>11</label>
                        </div>


                        <div className="product-element product-element--12-us-45-eu">
                            <input type="radio" name="single-option-select-1" id="product-1-12-us-45-eu"
                                   value="12 " data-index="option1" className="single-option-select" checked={this.state.Sizechecked === "12"}
                                   onChange={this.onValueChange}  />
                                <label htmlFor="product-1-12-us-45-eu" data-size="12" tabIndex={0}
                                       style={{ border: `${this.state.Sizechecked === "12" ? '2px solid black' : 'none'}` }}>12</label>
                        </div>


                        <div className="product-element selected-product product-element--13-us-46-eu">
                            <input type="radio" name="single-option-select-1" id="product-1-13-us-46-eu"
                                   value="13 (US) / 46 (EU)" data-index="option1" className="single-option-select"
                                   checked={this.state.Sizechecked === "13"}
                                   onChange={this.onValueChange} />
                                <label htmlFor="product-1-13-us-46-eu" data-size="13" tabIndex={0}
                                       style={{ border: `${this.state.Sizechecked === "13" ? '2px solid black' : 'none'}` }}>13</label>
                        </div>

                    </div>


                    <div id="product-purchase-row" className="product-option-row ">

                        <button name="add" id="add-to-cart" type="submit" className="btn btn--black btn--block"
                                aria-label="Add to Cart" onClick={this.AddtoCart }>
                                <span className="minicart-add-to-cart-text" id="add-to-cart-text"
                                      data-add-to-cart-text="Add To Cart">Add To Cart</span>
                        </button>

                    </div>
                    <div className="accordion" id="accordionExample">

                        <div className="card" style={{ border: 'none'}}>
                            <div className="card-header" id="headingOne">
                                <h2 className="mb-0">
                                    <button className="btn " type="button" data-toggle="collapse"
                                            data-target="#collapseOne" aria-expanded="true" aria-controls="collapseOne">
                                      Product Description
                                    </button>
                                </h2>
                            </div>

                            <div id="collapseOne" className="collapse show" aria-labelledby="headingOne"
                                 data-parent="#accordionExample">
                                <div className="card-body">
                                    Any look, any day. (Every day?) In buttery full-grain calf leather, the men’s Capri Triple White sneakers are the ultimate wardrobe cornerstone. The one that lets you do you. These White Capri shoes are so neutral they freshen and compliment just about anything. These sneakers let you reinvent the fashion wheel, or keep the white capri classic. Rather than canvas, these white low-top sneakers are made with real Italian leather for only the best men’s capri look and wearability. Keep these kicks  fresh or curiously banged up: your choice.
                                    Snag yourself a pair before you go out of style.
                                </div>
                            </div>
                        </div>
                    </div>




                </div>



                </div>


            </div>

        </div>

      </div>
    );
  }
}

export default withContext(GlobalContainer)(withRouter(CheckoutPage));
