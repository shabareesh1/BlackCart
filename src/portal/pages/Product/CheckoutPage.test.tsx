import React from "react";
import ReactDOM from "react-dom";
import CheckoutPage from "./CheckoutPage";
import "jest-styled-components";
import { MemoryRouter as Router } from "react-router-dom";
import { Provider } from "unstated";

import { render, Screen, cleanup } from "@testing-library/react";

describe("CheckoutPage", () => {
  afterEach(cleanup);

  test("should match snapshot", async () => {
    const { container } = render(
      <Provider>
        <Router>
          <CheckoutPage />
        </Router>
      </Provider>
    );
    expect(container).toMatchSnapshot();
  });
});
