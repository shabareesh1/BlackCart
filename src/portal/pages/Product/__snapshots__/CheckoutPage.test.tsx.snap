// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CheckoutPage should match snapshot 1`] = `
<div>
  <div>
    <div
      class="main  "
    >
      <div>
        <nav
          class="navbar navbar-expand-lg navbar-light bg-light"
        >
          <button
            aria-controls="navbarTogglerDemo03"
            aria-expanded="false"
            aria-label="Toggle navigation"
            class="navbar-toggler"
            data-target="#navbarTogglerDemo03"
            data-toggle="collapse"
            type="button"
          >
            <span
              class="navbar-toggler-icon"
            />
          </button>
          <a
            class="navbar-brand"
            href="#"
            style="font-weight: 700; font-size: 30px;"
          >
            KOIO
          </a>
          <div
            class="collapse navbar-collapse"
            id="navbarTogglerDemo03"
          >
            <ul
              class="navbar-nav mr-auto mt-2 mt-lg-0"
            >
              <li
                class="nav-item active"
              >
                <a
                  class="nav-link"
                  href="#"
                >
                  Shop men 
                  <span
                    class="sr-only"
                  >
                    (current)
                  </span>
                </a>
              </li>
              <li
                class="nav-item"
              >
                <a
                  class="nav-link"
                  href="#"
                >
                   Shop Women
                </a>
              </li>
              <li
                class="nav-item"
              >
                <a
                  class="nav-link "
                  href="#"
                >
                  About 
                </a>
              </li>
            </ul>
            <div
              class="form-inline my-2 my-lg-0"
            >
              <ul
                class="navbar-nav mr-auto mt-2 mt-lg-0"
              >
                <li
                  class="nav-item "
                >
                  <a
                    class="nav-link"
                    href="#"
                  >
                     Locations 
                    <span
                      class="sr-only"
                    >
                      (current)
                    </span>
                  </a>
                </li>
                <li
                  class="nav-item"
                >
                  <a
                    class="nav-link"
                    href="#"
                  >
                      CraftmanShip
                  </a>
                </li>
                <li
                  class="nav-item"
                >
                  <a
                    class="nav-link "
                    href="#"
                  >
                     Cart (
                    0
                    ) 
                  </a>
                </li>
              </ul>
            </div>
          </div>
        </nav>
      </div>
      <div
        class=" Content container-fluid"
      >
        <div
          class="row"
          style="height: 100%;"
        >
          <div
            class="col Productimages"
          >
            <div
              class=" row"
            >
              <div
                class="col"
              >
                <div>
                  <img
                    class="image"
                    src="./image1.jpg"
                  />
                </div>
                <div>
                  <img
                    class="image"
                    src="./image2.png.jpg"
                  />
                </div>
                <div>
                  <img
                    class="image"
                    src="./image3.png"
                  />
                </div>
                <div>
                  <img
                    class="image"
                    src="./image1.jpg"
                  />
                </div>
                <div>
                  <img
                    class="image"
                    src="./image2.png.jpg"
                  />
                </div>
              </div>
              <div
                class="col"
              >
                <img
                  class="imageStyle"
                  id="0"
                  src="./large1.png"
                />
                <img
                  class="imageStyle"
                  id="1"
                  src="./large2.png"
                />
                <img
                  class="imageStyle"
                  id="2"
                  src="./large3.png "
                />
              </div>
            </div>
          </div>
          <div
            class="col Productdetails"
          >
            <header
              id="product-header"
            >
              <span>
                Men's
              </span>
              <div>
                CAPRI TRIPLE WHITE
              </div>
              <span
                class="product-pricing"
              >
                <span
                  class="product-price"
                >
                  $268
                </span>
              </span>
            </header>
            <div
              class="product-option-row product-option-row--productes product-option-row--7 product-option-row--size "
            >
              <h5
                class="product-header"
              >
                Size:
              </h5>
              <div
                class="product-element product-element--7-us-40-eu"
              >
                <input
                  checked=""
                  class="single-option-select"
                  data-index="option1"
                  id="product-1-7-us-40-eu"
                  name="single-option-select-1"
                  type="radio"
                  value="7 "
                />
                <label
                  data-size="7"
                  for="product-1-7-us-40-eu"
                  style="border: 2px solid black;"
                  tabindex="0"
                >
                  7
                </label>
              </div>
              <div
                class="product-element product-element--8-us-41-eu"
              >
                <input
                  class="single-option-select"
                  data-index="option1"
                  id="product-1-8-us-41-eu"
                  name="single-option-select-1"
                  type="radio"
                  value="8 "
                />
                <label
                  data-size="8"
                  for="product-1-8-us-41-eu"
                  tabindex="0"
                >
                  8
                </label>
              </div>
              <div
                class="product-element product-element--9-us-42-eu"
              >
                <input
                  class="single-option-select"
                  data-index="option1"
                  id="product-1-9-us-42-eu"
                  name="single-option-select-1"
                  type="radio"
                  value="9 "
                />
                <label
                  data-size="9"
                  for="product-1-9-us-42-eu"
                  tabindex="0"
                >
                  9
                </label>
              </div>
              <div
                class="product-element product-element--10-us-43-eu"
              >
                <input
                  class="single-option-select"
                  data-index="option1"
                  id="product-1-10-us-43-eu"
                  name="single-option-select-1"
                  type="radio"
                  value="10 "
                />
                <label
                  data-size="10"
                  for="product-1-10-us-43-eu"
                  tabindex="0"
                >
                  10
                </label>
              </div>
              <div
                class="product-element product-element--11-us-44-eu"
              >
                <input
                  class="single-option-select"
                  data-index="option1"
                  id="product-1-11-us-44-eu"
                  name="single-option-select-1"
                  type="radio"
                  value="11 "
                />
                <label
                  data-size="11"
                  for="product-1-11-us-44-eu"
                  tabindex="0"
                >
                  11
                </label>
              </div>
              <div
                class="product-element product-element--12-us-45-eu"
              >
                <input
                  class="single-option-select"
                  data-index="option1"
                  id="product-1-12-us-45-eu"
                  name="single-option-select-1"
                  type="radio"
                  value="12 "
                />
                <label
                  data-size="12"
                  for="product-1-12-us-45-eu"
                  tabindex="0"
                >
                  12
                </label>
              </div>
              <div
                class="product-element selected-product product-element--13-us-46-eu"
              >
                <input
                  class="single-option-select"
                  data-index="option1"
                  id="product-1-13-us-46-eu"
                  name="single-option-select-1"
                  type="radio"
                  value="13 (US) / 46 (EU)"
                />
                <label
                  data-size="13"
                  for="product-1-13-us-46-eu"
                  tabindex="0"
                >
                  13
                </label>
              </div>
            </div>
            <div
              class="product-option-row "
              id="product-purchase-row"
            >
              <button
                aria-label="Add to Cart"
                class="btn btn--black btn--block"
                id="add-to-cart"
                name="add"
                type="submit"
              >
                <span
                  class="minicart-add-to-cart-text"
                  data-add-to-cart-text="Add To Cart"
                  id="add-to-cart-text"
                >
                  Add To Cart
                </span>
              </button>
            </div>
            <div
              class="accordion"
              id="accordionExample"
            >
              <div
                class="card"
              >
                <div
                  class="card-header"
                  id="headingOne"
                >
                  <h2
                    class="mb-0"
                  >
                    <button
                      aria-controls="collapseOne"
                      aria-expanded="true"
                      class="btn "
                      data-target="#collapseOne"
                      data-toggle="collapse"
                      type="button"
                    >
                      Product Description
                    </button>
                  </h2>
                </div>
                <div
                  aria-labelledby="headingOne"
                  class="collapse show"
                  data-parent="#accordionExample"
                  id="collapseOne"
                >
                  <div
                    class="card-body"
                  >
                    Any look, any day. (Every day?) In buttery full-grain calf leather, the men’s Capri Triple White sneakers are the ultimate wardrobe cornerstone. The one that lets you do you. These White Capri shoes are so neutral they freshen and compliment just about anything. These sneakers let you reinvent the fashion wheel, or keep the white capri classic. Rather than canvas, these white low-top sneakers are made with real Italian leather for only the best men’s capri look and wearability. Keep these kicks  fresh or curiously banged up: your choice. Snag yourself a pair before you go out of style.
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Navigation bar should match snapshot 1`] = `
<div>
  <div>
    <div
      class="main  "
    >
      <div>
        <nav
          class="navbar navbar-expand-lg navbar-light bg-light"
        >
          <button
            aria-controls="navbarTogglerDemo03"
            aria-expanded="false"
            aria-label="Toggle navigation"
            class="navbar-toggler"
            data-target="#navbarTogglerDemo03"
            data-toggle="collapse"
            type="button"
          >
            <span
              class="navbar-toggler-icon"
            />
          </button>
          <a
            class="navbar-brand"
            href="#"
            style="font-weight: 700; font-size: 30px;"
          >
            KOIO
          </a>
          <div
            class="collapse navbar-collapse"
            id="navbarTogglerDemo03"
          >
            <ul
              class="navbar-nav mr-auto mt-2 mt-lg-0"
            >
              <li
                class="nav-item active"
              >
                <a
                  class="nav-link"
                  href="#"
                >
                  Shop men 
                  <span
                    class="sr-only"
                  >
                    (current)
                  </span>
                </a>
              </li>
              <li
                class="nav-item"
              >
                <a
                  class="nav-link"
                  href="#"
                >
                   Shop Women
                </a>
              </li>
              <li
                class="nav-item"
              >
                <a
                  class="nav-link "
                  href="#"
                >
                  About 
                </a>
              </li>
            </ul>
            <div
              class="form-inline my-2 my-lg-0"
            >
              <ul
                class="navbar-nav mr-auto mt-2 mt-lg-0"
              >
                <li
                  class="nav-item "
                >
                  <a
                    class="nav-link"
                    href="#"
                  >
                     Locations 
                    <span
                      class="sr-only"
                    >
                      (current)
                    </span>
                  </a>
                </li>
                <li
                  class="nav-item"
                >
                  <a
                    class="nav-link"
                    href="#"
                  >
                      CraftmanShip
                  </a>
                </li>
                <li
                  class="nav-item"
                >
                  <a
                    class="nav-link "
                    href="#"
                  >
                     Cart (
                    0
                    ) 
                  </a>
                </li>
              </ul>
            </div>
          </div>
        </nav>
      </div>
      <div
        class=" Content container-fluid"
      >
        <div
          class="row"
          style="height: 100%;"
        >
          <div
            class="col Productimages"
          >
            <div
              class=" row"
            >
              <div
                class="col"
              >
                <div>
                  <img
                    class="image"
                    src="./image1.jpg"
                  />
                </div>
                <div>
                  <img
                    class="image"
                    src="./image2.png.jpg"
                  />
                </div>
                <div>
                  <img
                    class="image"
                    src="./image3.png"
                  />
                </div>
                <div>
                  <img
                    class="image"
                    src="./image1.jpg"
                  />
                </div>
                <div>
                  <img
                    class="image"
                    src="./image2.png.jpg"
                  />
                </div>
              </div>
              <div
                class="col"
              >
                <img
                  class="imageStyle"
                  id="0"
                  src="./large1.png"
                />
                <img
                  class="imageStyle"
                  id="1"
                  src="./large2.png"
                />
                <img
                  class="imageStyle"
                  id="2"
                  src="./large3.png "
                />
              </div>
            </div>
          </div>
          <div
            class="col Productdetails"
          >
            <header
              id="product-header"
            >
              <span>
                Men's
              </span>
              <div>
                CAPRI TRIPLE WHITE
              </div>
              <span
                class="product-pricing"
              >
                <span
                  class="product-price"
                >
                  $268
                </span>
              </span>
            </header>
            <div
              class="product-option-row product-option-row--productes product-option-row--7 product-option-row--size "
            >
              <h5
                class="product-header"
              >
                Size:
              </h5>
              <div
                class="product-element product-element--7-us-40-eu"
              >
                <input
                  checked=""
                  class="single-option-select"
                  data-index="option1"
                  id="product-1-7-us-40-eu"
                  name="single-option-select-1"
                  type="radio"
                  value="7 "
                />
                <label
                  data-size="7"
                  for="product-1-7-us-40-eu"
                  style="border: 2px solid black;"
                  tabindex="0"
                >
                  7
                </label>
              </div>
              <div
                class="product-element product-element--8-us-41-eu"
              >
                <input
                  class="single-option-select"
                  data-index="option1"
                  id="product-1-8-us-41-eu"
                  name="single-option-select-1"
                  type="radio"
                  value="8 "
                />
                <label
                  data-size="8"
                  for="product-1-8-us-41-eu"
                  tabindex="0"
                >
                  8
                </label>
              </div>
              <div
                class="product-element product-element--9-us-42-eu"
              >
                <input
                  class="single-option-select"
                  data-index="option1"
                  id="product-1-9-us-42-eu"
                  name="single-option-select-1"
                  type="radio"
                  value="9 "
                />
                <label
                  data-size="9"
                  for="product-1-9-us-42-eu"
                  tabindex="0"
                >
                  9
                </label>
              </div>
              <div
                class="product-element product-element--10-us-43-eu"
              >
                <input
                  class="single-option-select"
                  data-index="option1"
                  id="product-1-10-us-43-eu"
                  name="single-option-select-1"
                  type="radio"
                  value="10 "
                />
                <label
                  data-size="10"
                  for="product-1-10-us-43-eu"
                  tabindex="0"
                >
                  10
                </label>
              </div>
              <div
                class="product-element product-element--11-us-44-eu"
              >
                <input
                  class="single-option-select"
                  data-index="option1"
                  id="product-1-11-us-44-eu"
                  name="single-option-select-1"
                  type="radio"
                  value="11 "
                />
                <label
                  data-size="11"
                  for="product-1-11-us-44-eu"
                  tabindex="0"
                >
                  11
                </label>
              </div>
              <div
                class="product-element product-element--12-us-45-eu"
              >
                <input
                  class="single-option-select"
                  data-index="option1"
                  id="product-1-12-us-45-eu"
                  name="single-option-select-1"
                  type="radio"
                  value="12 "
                />
                <label
                  data-size="12"
                  for="product-1-12-us-45-eu"
                  tabindex="0"
                >
                  12
                </label>
              </div>
              <div
                class="product-element selected-product product-element--13-us-46-eu"
              >
                <input
                  class="single-option-select"
                  data-index="option1"
                  id="product-1-13-us-46-eu"
                  name="single-option-select-1"
                  type="radio"
                  value="13 (US) / 46 (EU)"
                />
                <label
                  data-size="13"
                  for="product-1-13-us-46-eu"
                  tabindex="0"
                >
                  13
                </label>
              </div>
            </div>
            <div
              class="product-option-row "
              id="product-purchase-row"
            >
              <button
                aria-label="Add to Cart"
                class="btn btn--black btn--block"
                id="add-to-cart"
                name="add"
                type="submit"
              >
                <span
                  class="minicart-add-to-cart-text"
                  data-add-to-cart-text="Add To Cart"
                  id="add-to-cart-text"
                >
                  Add To Cart
                </span>
              </button>
            </div>
            <div
              class="accordion"
              id="accordionExample"
            >
              <div
                class="card"
              >
                <div
                  class="card-header"
                  id="headingOne"
                >
                  <h2
                    class="mb-0"
                  >
                    <button
                      aria-controls="collapseOne"
                      aria-expanded="true"
                      class="btn "
                      data-target="#collapseOne"
                      data-toggle="collapse"
                      type="button"
                    >
                      Product Description
                    </button>
                  </h2>
                </div>
                <div
                  aria-labelledby="headingOne"
                  class="collapse show"
                  data-parent="#accordionExample"
                  id="collapseOne"
                >
                  <div
                    class="card-body"
                  >
                    Any look, any day. (Every day?) In buttery full-grain calf leather, the men’s Capri Triple White sneakers are the ultimate wardrobe cornerstone. The one that lets you do you. These White Capri shoes are so neutral they freshen and compliment just about anything. These sneakers let you reinvent the fashion wheel, or keep the white capri classic. Rather than canvas, these white low-top sneakers are made with real Italian leather for only the best men’s capri look and wearability. Keep these kicks  fresh or curiously banged up: your choice. Snag yourself a pair before you go out of style.
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
